crates/jit/src/instantiate.rs

```diff
@@ -103,6 +105,22 @@ impl<'data> RawCompiledModule<'data> {
         // Make all code compiled thus far executable.
         compiler.publish_compiled_code();
 
+        // Initialize profiler and load the wasm module
+        match profiler {
+            Some(_) => {
+                let region_name = String::from("wasm_module");
+                let mut profiler_new = profiler.unwrap().lock().unwrap();
+                profiler_new.init().ok();
```

Comment:
I wonder what will happen when multiple instances are initialized. Can we make the `init` internal to JitDumpAgent and call it once in the `module_load`?

Also we are ignoring error here, shall we just `drop()` instead of `.ok()`?

Reply:
The latest push we make the call to init internal where we check the ok() to decide if we need to continue with the module load or print a message of failure:

https://github.com/bytecodealliance/wasmtime/pull/360/files#diff-5550f3278ee1a915f45b51e1fa707e1cR653-R657

Does this seem reasonable? This way .. for what ever reason the initial failure, if the call is tried again it still has a chance to work and start profiling from that point on.

Reply:
It is reasonable. Since init() is a prerequisite and part of the module_load() logic, can we make init() internal and call it in the module_load()?

Reply:
okay, it done. 👍